### ISSUE TITLE
CORB blocking new livechat version

### ISSUE THREAD

**Issue by RealKelsar** [RocketChat/Rocket.Chat#16002]
### Description:

Hi, we got the same Problem as in #15545, but removing the Version did not help. We also overwrote the Mimetype, didn't help either. Since i can't add there, here is a new Issue.

As far as I understood https://www.chromium.org/Home/chromium-security/corb-for-developers it sees .js and checks the Mimetype, which is text/html, which i should be, since the content is html, as far as I see it.

I am currently out of Ideas, so some Hint would be appreciated.

Error:

```
Cross-Origin Read Blocking (CORB) blocked cross-origin response https://example.com/packages/rocketchat_livechat/assets/rocketchat-livechat.min.js with MIME type text/html. See https://www.chromestatus.com/feature/5629709824032768 for more details.
```

**Comment by renatobecker** [comment]
Hey @RealKelsar,

Certainly, there is something wrong with the Javascript snippet installation code on the client where the Livechat widget is installed.

Did you update the installation code already?

**Comment by RealKelsar** [comment]
Hi,

I updated RocketChat to the newest Version and replaced the code, but did not really see a difference in the Code, the old one was from the 2.2.0 Version I think. But neverless that worked, kind of.

Had a 2nd problem, because I had my Browser set to Block 3rd Party Cookies, after removing that it works now.

I have a http/2 error left, but I worked around that by disabling http/2 on the RocketChat Server for now. Will investigate that later.

Thank you.
